src/services/users.js

```diff
@@ -24,3 +24,23 @@ export async function logout() {
   const response = await client.auth.signOut();
   return checkError(response);
 }
+
+export async function createToDo(task) {
+  const resp = await client.from('todos').insert([{ task: task, user_id: client.auth.user().id }]);
+  return checkError(resp);
+}
+
+export async function getToDos() {
+  const resp = await client.from('todos').select('*');
```

Comment:
I might add a `.order()` after this so your todos always come back in the same order